vllm/model_executor/layers/sampler.py

```diff
@@ -13,6 +13,10 @@
 
 _SAMPLING_EPS = 1e-5
 
+# For each sequence group, we generate a list of SequenceOutputs object,
+# each of which contains one possible candidate for the next token.
+SamplerOutput = List[List[SequenceOutputs]]
```

Comment:
Would it be possible to define this type in `sequence.py` or anywhere else? I feel it's a bit weird that `llm_engine.py` imports `sampler.py`. This could make the dependencies between the files a bit unclear and require the process running `LLMEngine` to unnecessarily import `torch`.